contexts/v1/index.json

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+    "Eip712SchemaValidator2021": "https://w3id.org/security#Eip712SchemaValidator2021",
+    "EthereumEip712Signature2021": {
+      "@id": "https://w3id.org/security#EthereumEip712Signature2021",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "created": {
+          "@id": "http://purl.org/dc/terms/created",
+          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+        },
+        "domain": "https://w3id.org/security#domain",
+        "proofPurpose": {
+          "@id": "https://w3id.org/security#proofPurpose",
+          "@type": "@vocab",
+          "@context": {
+            "@version": 1.1,
+            "@protected": true,
+            "id": "@id",
+            "type": "@type",
+            "assertionMethod": {
+              "@id": "https://w3id.org/security#assertionMethod",
+              "@type": "@id",
+              "@container": "@set"
+            },
+            "authentication": {
+              "@id": "https://w3id.org/security#authenticationMethod",
+              "@type": "@id",
+              "@container": "@set"
+            }
```

Comment:
Does this mean we are not supporting other proof purposes? This signature suite might be also used with ZCAPs potentially.

Reply:
Yes, good point... added capabilityInvocation, capabilityDelegation, and keyAgreement in 4504e4a765041da7437c29d0c498aa64eebdc9a6; matching what is used in https://github.com/w3c-ccg/lds-jws2020

Reply:
let's remove key agreement. i don't see how this might be able. the other stuff looks good! thx!